be/src/exprs/bitmap_functions.cpp

```diff
@@ -675,4 +675,55 @@ StatusOr<ColumnPtr> BitmapFunctions::bitmap_subset_in_range(FunctionContext* con
     return builder.build(ColumnHelper::is_all_const(columns));
 }
 
+StatusOr<ColumnPtr> BitmapFunctions::bitmap_to_binary(FunctionContext* context, const starrocks::Columns& columns) {
+    ColumnViewer<TYPE_OBJECT> viewer(columns[0]);
+
+    size_t size = columns[0]->size();
+    ColumnBuilder<TYPE_VARCHAR> builder(size);
+
+    std::unique_ptr<char[]> buf;
+    size_t cur_buf_size = 0;
+
+    for (int row = 0; row < size; ++row) {
+        BitmapValue* bitmap = viewer.value(row);
+        int serialize_size = bitmap->getSizeInBytes();
+        if (serialize_size > cur_buf_size) {
+            buf.reset(new char[serialize_size]);
+            cur_buf_size = serialize_size;
+        }
+
+        bitmap->write((char*)buf.get());
+        builder.append(Slice(buf.get(), serialize_size));
+    }
+    ColumnPtr col = builder.build(ColumnHelper::is_all_const(columns));
+    if (col->capacity_limit_reached()) {
+        return Status::InternalError("Size of binary column generated by bitmap_to_binary reaches limit (4G)");
+    } else {
+        return col;
+    }
+}
+
+StatusOr<ColumnPtr> BitmapFunctions::bitmap_from_binary(FunctionContext* context, const Columns& columns) {
+    ColumnViewer<TYPE_VARCHAR> viewer(columns[0]);
+    size_t size = columns[0]->size();
+    ColumnBuilder<TYPE_OBJECT> builder(size);
+
+    for (int row = 0; row < size; ++row) {
+        if (viewer.is_null(row)) {
+            builder.append_null();
+            continue;
+        }
+
+        auto src_value = viewer.value(row);
+        if (src_value.size == 0) {
+            builder.append_null();
+            continue;
+        }
+
+        BitmapValue bitmap(src_value.data);
```

Comment:
what if the input is not a valid format? should we consider to report error?

Reply:
Good suggestion, i will add the check.